packages/cerebral-website-tutorial/parts/05_update_state/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Controller } from 'cerebral'
+import App from './components/App'
+import { Container } from 'cerebral/react'
+import Devtools from 'cerebral/devtools'
+import { set } from 'cerebral/operators'
```

Comment:
@fopsdev no need in eject and new npm release as well. just run `npm run setup` at the root of monorepo and it will link built cerebral to your package.